index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+    <link rel="stylesheet" href="index.css">
+    <title>Portifolio page revire</title>
+</head>
+
+<body>
+    <nav id="navbar">
+        <img id="logo" class="dagi" width='150'
+            src="https://avatars.githubusercontent.com/u/68595873?s=400&u=80bbdfc291f01cffaae57bee393b989ea3b24bda&v=4"
+            alt="">
+
+        <h1 class="float-left center">
+            Dagmawi zewdu
+        </h1>
+        <p class="center">
+            <a href="#dagic" class="white">
+                <span class="iconify" data-icon="mdi:github"></span>
+                dagi.zewdu.dz@gmail.com
+            </a>
+        </p>
+
+    </nav>
+    <section id="welcome-section">
+        <h1 class="welcome center">
+            Welcome
+        </h1>
+        <h3 class="center">hey i am Dagic, I am full stack developer</h3>
+    </section>
+    <section id="projects">
+    <h2 class="center">Projects</h2>
+        <div class="projects">
+            <div class="project-tile">
+                <a href="https://github.com/Dagic-zewdu/webook">
+
+                    <h1 class="center">Webook</h1>
+                </a>
+
+            </div>
+
+            <div class="project-tile">
+                <a href="https://github.com/Dagic-zewdu/csa_client">
+
+                    <h1 class="center">Allowance system</h1>
+                </a>
+
+            </div>
+
+            <div class="project-tile">
+                <a href="https://github.com/Rehubot/moct-web">
+
+                    <h1 class="center">Ministry of culture and tourism</h1>
+                </a>
+
+            </div>
+            <div class="project-tile">
+                <a href="https://github.com/evans22j/evans-moise">
+
+                    <h1 class="center">Evans moise</h1>
+                </a>
+            </div>
+
+
+            <div class="project-tile">
+                <a href="https://github.com/Rehubot/moct-web">
+                    <h1 class="center">HTML</h1>
+                </a>
+
+            </div>
+            <div class="project-tile">
+                <a href="https://github.com/evans22j/link">
+
+
+                    <h1 class="center">Link</h1>
+
+                </a>
```

Comment:
Remove uncessarilly white spaces

Reply:
It would be super helpful if you always give an explanation for your requirements. One of the many objectives code reviews have is to help the person being reviewed, and what better way to accomplish this than to give them a clear reason on why we are suggesting the changes 🤓 Kindly explain in your comments the reason for your changes 😎

Also, please be sure to tell your requirements/suggestions in a way that it sounds like suggestions instead of orders 😉 The idea is to make it politely and friendly 😬